ics/serializers/icalendar_serializer.py

```diff
@@ -3,6 +3,9 @@
 
 
 class CalendarSerializer(Serializer):
+    def serialize_0version(calendar, container):  # 0version will be sorted first
+        container.append(ContentLine("VERSION", value="2.0"))
+
     def serialize_prodid(calendar, container):
```

Comment:
```suggestion
    def serialize_1prodid(calendar, container): # 1prodid will be sorted second
```

Otherwise, it's good for me :)